samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.12",
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
```

Comment:
Please remove duplicate sample

Reply:
PR link for removing duplicate sample
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/723